.pax/prepare-workspace.sh

```diff
@@ -23,6 +23,7 @@ set -x
 
 # expected workspace layout:
 # ./.pax/mediation/
+# ./.pax/keyring-util/
```

Comment:
keep 'm in alphabetical order so it lines up with output of ls

Reply:
Done.